src/cmds/qstat.c

```diff
@@ -309,6 +309,47 @@ states(char *string, char *q, char *r, char *h, char *w, char *t, char *e, int l
 	}
 }
 
+/**
+ * @brief
+ *	replace special characters such as newline and form feed with "\\n" and "\\f"
+ *
+ * @param[in]   val - string which needs to be formatted
+ *
+ * @return string
+ * @retval NULL     - if memory allocation failed.
+ * @retval string   - string in which special characters are replaced.
+ *
+ */
+char *
+replace_special_chars(char *val) {
```

Comment:
function name is misleading... This function is prepending an escape char before new line and form feed.